README.md

```diff
@@ -273,3 +271,26 @@ By overriding it you can change each substitution token. Such as:
 * Change the `__name__` that appears in the untranslated `msgid` text.
 * Change the `key` that is assigned to React elements.
 * Change the `value` by injecting props, converting text to elements, etc.
+
+### `splitPlural : function`
+
+This merely implements the split of `msgid` for [ngettext](https://linux.die.net/man/3/ngettext).
+
+Here is the default implementation.
+
+```javascript
+const splitPlural = msgid => msgid.split('|');
+splitPlural.expect = 2;
+```
+
+If you omit `expect` count there will be no validation (during development) of the number of delimiters in the template string.
+
+The default implementation is supports exactly 2 plural forms.
+
+### `isProduction : boolean`
```

Comment:
Common practice is to use `process.env.NODE_ENV === 'production'` and also do production-ready minified build.